Docs runtime image override (#13895)
kyma-project/kyma: docs/01-overview/main-areas/serverless/svls-06-security-considerations.md

To eliminate potential security risks when using Functions, bear in mind these few facts:

> - Kyma provides base images for serverless runtimes. Those default runtimes are maintained with regards commonly known security advisories. It is possible to use a custom runtime image (check the [tutorial](../../../03-tutorials/00-serverless/svls-13-override-runtime-image.md)), however in such case the responsibility of security compliance and assesing exploitability of any potential vulnerabilities of the custom runtime image remains on the user side.

Reviewer: ```suggestion
- Kyma provides base images for serverless runtimes. Those default runtimes are maintained with regards to commonly known security advisories. It is possible to use a custom runtime image (see this [tutorial](../../../03-tutorials/00-serverless/svls-13-override-runtime-image.md)). In such a case, you are responsible for security compliance and assessment of exploitability of any potential vulnerabilities of the custom runtime image.
```
